### PR TITLE
completed dom-querying

### PR DESCRIPTION
**Pull request by Logisticx**
image

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+console.log('hello, world');
+console.log(document.querySelector('h1'));
```
Comment (Zhiwutian) [inline dom-querying/main.js:2]: there should be console.dir calls for each of these as well

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
make sure to include the console.dir calls in your code. Reach out if you need clarity!

**Comment by Logisticx** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**